samples/openapi3/client/petstore/python-experimental/petstore_api/configuration.py

```diff
@@ -396,15 +396,15 @@ def logger_format(self, value):
         self.__logger_format = value
         self.logger_formatter = logging.Formatter(self.__logger_format)
 
-    def get_api_key_with_prefix(self, identifier):
+    def get_api_key_with_prefix(self, identifier, alias=None):
```

Comment:
How about changing alias to `identifier_alias` to help convey that this is related to the identifier?

For the key assignment how about something like this easier to understand snippet?
```
key = self.api_key.get(identifier)
if key is None and identifier_alias is not None:
  key = self.api_key.get(alias)
```

Reply:
I would really love to remove this internal function from public API.

Reply:
I will add the docstring, but `identifier_alias` seems unnecessary too long.